documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -880,6 +881,14 @@ factory method defined in the `Arguments` interface.
 include::{testDir}/example/ParameterizedTestDemo.java[tags=multi_arg_MethodSource_example]
 ----
 
+External _factory_ method can be referenced by providing _fully qualified method name_ as
+demonstrated in the following example.
```

Comment:
Please change to:

An external, `static` _factory_ method can be referenced by providing its _fully qualified method name_ as
+demonstrated in the following example.